alchemy/src/cloudflare/worker-metadata.ts

```diff
@@ -295,16 +296,19 @@ export async function prepareWorkerMetadata<B extends Bindings>(
             : [],
       ),
     ],
-    migrations: {
-      new_classes: props.migrations?.new_classes ?? [],
-      deleted_classes: [
-        ...(deletedClasses ?? []),
-        ...(props.migrations?.deleted_classes ?? []),
-      ],
-      renamed_classes: props.migrations?.renamed_classes ?? [],
-      transferred_classes: props.migrations?.transferred_classes ?? [],
-      new_sqlite_classes: props.migrations?.new_sqlite_classes ?? [],
-    },
+    // Skip migrations for Workers for Platform as they are not supported
```

Comment:
This is not true. 

https://developers.cloudflare.com/api/resources/workers_for_platforms/subresources/dispatch/subresources/namespaces/subresources/scripts/methods/update/

It has also made me realize another bug with dispatch namespaces. There are different endpoints for uploading assets and basically every api has a workers for platforms variant. 

We need to update all the other code involved with workers for platforms workers, durable objects, workflows whatever 

Take a look at the docs I sent with the dispatch namespaces. Make sure you've seen all of the wfp api docs and go ahead and update the codebase.